### ISSUE TITLE
[Tabs] Is .tabs-title's "font-size" and "color" on the a tag needed?

### ISSUE THREAD

**Issue by andrewbiddinger** [zurb/foundation-sites#7675]
I would like to make a case to remove .tabs-title's "font-size" and "color." Instead, I think it should resemble .menu's structure: display, padding, line-height.

/* Old */
.tabs-title > a {
    display: block;
    padding: 1.25rem 1.5rem;
    line-height: 1;
    font-size: 12px;
    color: #2199e8;
}

/* Structure Example */
.menu>li:not(.menu-text)>a {
    display: block;
    padding: 0.7rem 1rem;
    line-height: 1;
}

/* Proposed Change */
.tabs>li:not(.tabs-text)>a {
    display: block;
    padding: 0.7rem 1rem;
    line-height: 1;
}

.tabs-text {
    font-weight: bold;
    color: inherit;
    line-height: 1;
    padding: 0.7rem 1rem;
}

**Comment by andrewbiddinger** [comment]
Any thoughts on this one?

**Comment by gakimball** [comment]
We looked over this, here are our suggestions:
- Add a variable for `$tab-item-font-size` that uses the `12px` value currently hardcoded.
- Drop the `$tab-item-color` variable, in favor of inheriting the default link color. We like less settings and styles!

What do you think?

**Comment by andrewbiddinger** [comment]
That is sounding good, @gakimball . This may be said in ignorance, I still want to question the use of 12px, but let me know if it distrupts a grand scheme. 

My use case is having a secondary tab menu in the .off-canvas .menu. The regular menu doesn't specify what size the a or li tags are, it's all inherited. So, the .tabs-title doesn't receive the inherited size and I have to specically overide them. What's the reasoning behind specifying it?

Thanks!

**Comment by andycochran** [comment]
Drop `$tab-item-color`. :+1: 

How 'bout we don't size tab font at all? Do so with app-specific styles. 

Or, if we want to provide options for tab font sizes, leverage `$button-sizes` the way button groups do:
```
<ul class="tabs [tiny,small,large]" data-tabs id="example-tabs">
```
(I know we like to keep components isolated. But importing `components/button-group` depends on `components/button`, and tabs are basically buttons.)

**Comment by andrewbiddinger** [comment]
Drop ```$tab-item-color``` :+1: 

I like the idea of ```[tiny,small,large]``` and think that could be a good solution.

Although, I think of ```.tabs-title``` being more in the family of ```.menu``` (which has no specific sizing). That is why I think it should resemble it's "cousin" ```.menu```. 

Maybe ```.menu``` should have ```[tiny,small,large]``` as well? Or, should both be inherited?

**Comment by andycochran** [comment]
Good point, @andrewbiddinger. This makes me wonder if Foundation should instead have a global font sizes map… 
```scss
$font-sizes: (
  tiny: 0.6rem,
  small: 0.75rem,
  default: 1rem,
  large: 1.25rem,
) !default;
```
…much like `$button-sizes` (maybe instead of). 

Then various components—menus, buttons, breadcrumbs, etc.—could reference this map. This would have the added benefit of reducing the number of font sizes in your design. More than a handful is too many, and Foundation already uses these `px` font sizes: 48, 40, 31, 25, 24, 20, 19, 18, 17, 16, 14.4, 14, 13, 12.8, 12, 11, and 9.6.

**Comment by gakimball** [comment]
@andycochran We had some internal chats about this. It's probably worth our time to audit our use of font sizes so they're consistent. We did the same thing for colors, using one black, one white, and three grays.

That being said, we won't necessarily expose a font sizes map to our users. If you're implementing a visual design, you probably have a wide array of font sizes anyway, so a map isn't necessarily going to help you.

**Comment by gakimball** [comment]
Implemented the tab variable changes in the `6.2` branch: 367ad63270eb39d602014f8f35d02b60debb705c

We'll also look into auditing our font sizes in the base styles.

**Comment by andycochran** [comment]
Awesome, @gakimball. Limiting the number of font sizes is a good idea. If someone is implementing design, hopefully they _don't_ have a "wide array of font sizes." But that doesn't have to be done with a map.
